mitreattack/attackToExcel/stixToDf.py

```diff
@@ -7,6 +9,12 @@
 import numpy as np
 
 
+Matrix_Platforms_Lookup = {"enterprise-attack": ['PRE', 'Windows', 'macOS', 'Linux', 'Cloud', 'Office 365', 'Azure AD',
```

Comment:
As this is a constant I believe [it should be in all caps](https://www.python.org/dev/peps/pep-0008/#constants) instead of capitalizing just the first character of every word.

Reply:
Can do.